avaframe/com1DFA/com1DFACfg.ini

```diff
@@ -417,7 +417,7 @@ contourResType = ppr
 # thresholdValue for contour lines plot
 thresholdValue = 1.
 # create data required for range-time diagram of simulation results -
-# additional configurations in ana5Utils - distanceTimeAnalysisCfg
+# additional configurations in section distanceTimeAnalysis_override
```

Comment:
Should we point to a general information about override sections?

Reply:
now pointing again to ana5Utils/distanceTimeAnalysisCfg or its locl copy